javascript_ts/11.qnamaker/qnamaker.bot

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "qnamaker",
+    "secretKey": "",
+    "services": [
+        {
+            "type": "endpoint",
+            "name": "qnamaker",
+            "id": "http://localhost:3978/api/messages",
+            "appId": "",
+            "appPassword": "",
+            "endpoint": "http://localhost:3978/api/messages"
+        },
+        {
```

Comment:
Please make sure that having these blank sections in the .bot file will actually work with MSBOT CLI.